code/datums/status_effects/buffs.dm

```diff
@@ -323,25 +323,116 @@
 /datum/status_effect/fleshmend
 	id = "fleshmend"
 	alert_type = /atom/movable/screen/alert/status_effect/fleshmend
-	duration = 100
+	//Actual healing lasts for 30 seconds
+	duration = 320
+	var/ticks_passed = 0
 
 /datum/status_effect/fleshmend/tick()
+	ticks_passed ++
 	if(owner.on_fire)
 		linked_alert.icon_state = "fleshmend_fire"
 		return
 	else
 		linked_alert.icon_state = "fleshmend"
-	owner.adjustBruteLoss(-2.5, FALSE)
-	owner.adjustFireLoss(-2.5, FALSE)
-	owner.adjustOxyLoss(-2.5)
-	owner.adjustCloneLoss(-2.5)
-	owner.adjustToxLoss(-2.5, FALSE, TRUE)
+	if(ticks_passed < 2)
+		return
+	else if(ticks_passed == 2)
```

Comment:
?
prob(80) moment

Reply:
if anything make it prob chance instead of whatever is this

Reply:
This means the effect does nothing for the first 2 ticks (first 2 seconds) before the healing kicks in.
It creates a 2 second buffer between activating the ability and it having an effect